From c432d27f57e628f439a54883e5bc1656543b783d Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Wed, 18 Jan 2023 17:09:14 +0800
Subject: [PATCH] Revert "dmaengine: idxd: Fix crc_val field for completion
 record"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 490b233677de945349c700a552fac23be8b754eb.
---
 include/uapi/linux/idxd.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/idxd.h b/include/uapi/linux/idxd.h
index f086c5579006..9d9ecc0f4c38 100644
--- a/include/uapi/linux/idxd.h
+++ b/include/uapi/linux/idxd.h
@@ -188,7 +188,7 @@ struct dsa_completion_record {
 		};
 
 		uint32_t	delta_rec_size;
-		uint64_t	crc_val;
+		uint32_t	crc_val;
 
 		/* DIF check & strip */
 		struct {
-- 
2.35.3

